cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -1318,11 +1310,15 @@ SwiftGenerator::writeMarshalAsyncOutParams(::IceInternal::Output& out, const Ope
     // Marshal parameters in this order '(required..., optional...)'.
 
     out << nl << "let " << operationReturnDeclaration(op) << " = value";
-    for (const auto& param : op->sortedReturnAndOutParameters("returnValue"))
+    for (const auto& param : op->sortedReturnAndOutParameters(getEscapedParamName(op->outParameters(), "returnValue")))
     {
-        // 'isOutParam' fails for return types, and for return types, we don't want the 'mappedName'.
```

Comment:
I guess this was about the xxx:identifier metadata on the operation?

This is now fixed, so we can use mappedName all we want.